lib/pages/chat/chat.dart

```diff
@@ -560,7 +568,20 @@ class ChatController extends State<Chat>
     }
   }
 
-  void _handleDownloadFileForPreview({required Event event}) async {
+  void _handlePreviewWeb({required Event event}) async {
+    if (!event.hasAttachment) {
+      TwakeSnackBar.show(context, L10n.of(context)!.errorPreviewingFile);
+      return;
+    }
+
+    if (event.isPdfFile()) {
+      return previewPdfWeb(context, event);
+    }
```

Comment:
how about image and video?

Reply:
I tested it on the branch. It works well because When user clicks on msg we check `MessageTypes.File`.

<img width="467" alt="Screenshot 2023-10-06 at 12 00 08" src="https://github.com/linagora/twake-on-matrix/assets/99852347/a0d73284-22a6-49bd-83d5-a9dd1e9d7cf4">

Reply:
how about the video: 
<img width="611" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/6462404/6452de62-2a55-4baf-b93b-5515369eb7d5">

Reply:
It's fine

Reply:
let check mobile